src/tests/electron/common/view-controllers/left-nav-controller.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { needsReviewTestConfig } from 'electron/platform/android/test-configs/needs-review/test-config';
+import { TestConfig } from 'electron/types/test-config';
+import { leftNavAutomationId } from 'electron/views/left-nav/left-nav';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { SpectronAsyncClient } from 'tests/electron/common/view-controllers/spectron-async-client';
+
+export class LeftNavController {
+    private leftNavSelector = getAutomationIdSelector(leftNavAutomationId);
+
+    constructor(private readonly client: SpectronAsyncClient) {}
+
+    public async clickNeedsReview(): Promise<void> {
+        await this.clickLeftNavItem(needsReviewTestConfig);
+    }
+
+    private async clickLeftNavItem(config: TestConfig): Promise<void> {
+        const selector = this.getSelectorForLeftNavItemLink(config);
+        await this.client.click(selector);
```

Comment:
It looks like there's some flakiness happening here in the e2e tests at this point, where the tests are trying to click on a selector that doesn't exist yet. I think it would probably help to include a `this.waitForSelector(selector)` before the click.